GunplaOnlineShop/GunplaOnlineShop/ViewModels/RegisterViewModel.cs

```diff
@@ -9,11 +9,13 @@ namespace GunplaOnlineShop.ViewModels
     public class RegisterViewModel
     {
         [Required]
+        [Display(Name = "Email Address")]
         [DataType(DataType.EmailAddress)]
         [EmailAddress]
         public string Email { get; set; }
 
         [Required]
+        [Display(Name = "Password")]
```

Comment:
if you want to display the text the same as the property name, no need to use `[Display(Name = "Password")]`, by default, it will display as the property name  in `<label asp-for="xxx">`